fineract-provider/src/main/resources/sql/migrations/core_db/V365__reportCategoryList-FINERACT-1306.sql

```diff
@@ -18,5 +18,10 @@
 --
 
 -- two tables added: ReportCategoryList and FullReportList (FINERACT-1306)
-INSERT INTO stretchy_report (report_name, report_type, report_category, report_sql, description, core_report, use_report)VALUES ("ReportCategoryList", 'Table', '(NULL)', '(NULL)', '(NULL)', 1, 1);
-INSERT INTO stretchy_report (report_name, report_type, report_category, report_sql, description, core_report, use_report)VALUES ("FullReportList", 'Table', '(NULL)', '(NULL)', '(NULL)', 1, 1);
+INSERT INTO stretchy_report (report_name, report_type, report_category, report_sql, description, core_report, use_report)
```

Comment:
I think this duplicates another PR that was fixing the issue with RunReports... my suggestion would be to pull this to a separate PR rather than combine with the SQL parameterisation.

Reply:
Agree. Will undo this change. When the runreport fix split PR (inline comment 3) is merged the developer there would create undo script for this.

I will let you know when the changes are done. 
Thank you for your feedback!